src/cf-forms.less

```diff
@@ -448,6 +456,102 @@ textarea {
 
 }
 
+/* topdoc
+  name: Range input
+  family: cf-forms
+  patterns:
+    - name: Range slider input
+      markup: |
+        <label for="range-slider">Range Slider</label>
+        <input id="range-slider" type="range" min="0" max="100">
+  tags:
+    - cf-forms
+*/
+
+input[type=range] {
+    width: 100%;
+    padding-left: 0;
+    margin: 15px 0;
+    -webkit-appearance: none;
+    border: none;
+    &::-webkit-slider-runnable-track {
+        width: 100%;
+        height: 11px;
+        background: @input-range-bg;
+        border: 1px solid @input-range-border;
+        border-radius: 8px;
+        cursor: pointer;        
+    }
+    &::-moz-range-track {
+        width: 100%;
+        height: 11px;
+        background: @input-range-bg;
+        border: 1px solid @input-range-border;
+        cursor: pointer;
+    }
+    &::-ms-track {
+        width: 100%;
```

Comment:
While I was refactoring the Planning for Retirement slider to use this, I had to add a couple of styles to `&::-ms-track` to keep the top and button of the thumb from getting cut off in IE:

```css
border-width: 19px 0;
border-color: transparent;
```